src/__tests__/components/SideMenu/SideMenu.test.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import '@testing-library/jest-dom/extend-expect';
+import renderer from 'react-test-renderer';
+
+import SideMenu from '../../../components/SideMenu/SideMenu.component';
+
+let tree;
```

Comment:
Same comment as above. Remove this shared variable and create a wrapper per unit test